license-maven-plugin/src/main/java/com/mycila/maven/plugin/license/header/HeaderParser.java

```diff
@@ -167,11 +167,11 @@ private boolean hasHeader() {
             boolean foundEnd = false;
             do {
                 inPlaceHeader.append(line.toLowerCase());
-                line = fileContent.nextLine();
```

Comment:
If `end tag` is on the current line, it'll be missed and the whole file will be treated as the header